translations/tools/manage.js

```diff
@@ -202,17 +212,28 @@ program
       sourceLanguage
     } = config
 
-    if (targetFormat === 'csv') {
-      if (path.extname(fromPath) === '.json') {
+    const sourceFormat = path.extname(fromPath).replace('.', '') || 'csv'
+
+    const converter = new Map([
+      ['json-csv', () => {
+        if (!fs.existsSync(toPath)) {
+          fs.mkdirSync(toPath)
+        }
         const json = JSON.parse(fs.readFileSync(fromPath, 'utf8'))
         writeCsvFromJson(json, toPath, sourceLanguage, Object.keys(supportedLanguages))
-      } else {
-        throw new Error('nyi')
-      }
-    } else if (targetFormat === 'json') {
-      writeJsonFromCsv(fromPath, toPath, sourceLanguage)
+      }],
+      ['csv-json', () => {
+        writeJsonFromCsv(fromPath, toPath, sourceLanguage)
+      }]
+    ])
+
+    const convert = converter.get(`${sourceFormat.toLowerCase()}-${targetFormat.toLowerCase()}`)
```

Comment:
Sweet, didn't know they have Map in ES6 until now

Reply:
Also use them with caution. This is a bad use it it actually :D first wanted to make the key an array. But it does not work.

Will revert this to a plain object.